src/applications/pokemon_summary_screen/subscreen.c

```diff
@@ -24,30 +24,30 @@ static void sub_0223D0BC(PokemonSummaryScreen *summaryScreen);
 static int CheckSubscreenRectanglePress(PokemonSummaryScreen *summaryScreen);
 
 static const PSSSubscreenButton sSubscreenButtons_Normal[] = {
-    { PSS_PAGE_INFO, 0x1, 0x1, 0x4 },
-    { PSS_PAGE_MEMO, 0x1, 0x2, 0xA },
-    { PSS_PAGE_SKILLS, 0x2, 0x5, 0xF },
-    { PSS_PAGE_BATTLE_MOVES, 0x3, 0xA, 0x12 },
-    { PSS_PAGE_CONDITION, 0x2, 0x11, 0x12 },
-    { PSS_PAGE_CONTEST_MOVES, 0x3, 0x16, 0xF },
-    { PSS_PAGE_RIBBONS, 0x4, 0x19, 0xA },
-    { PSS_PAGE_EXIT, 0x4, 0x1A, 0x4 },
-    { PSS_PAGE_NONE, 0xFF, 0xFF, 0xFF }
+    { PSS_PAGE_INFO, 1, 1, 4 },
+    { PSS_PAGE_MEMO, 1, 2, 10 },
+    { PSS_PAGE_SKILLS, 2, 5, 15 },
+    { PSS_PAGE_BATTLE_MOVES, 3, 10, 18 },
+    { PSS_PAGE_CONDITION, 2, 17, 18 },
+    { PSS_PAGE_CONTEST_MOVES, 3, 22, 15 },
+    { PSS_PAGE_RIBBONS, 4, 25, 10 },
+    { PSS_PAGE_EXIT, 4, 26, 4 },
+    { PSS_PAGE_NONE, 255, 255, 255 }
```

Comment:
I would personally use `-1` for these max-value entries to more clearly signify that the value should not be trusted or used.